src/Plugin/GraphQL/DataProducer/Field/EntityReferenceRevisions.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace Drupal\graphql\Plugin\GraphQL\DataProducer\Field;
+
+use Drupal\Core\Cache\RefinableCacheableDependencyInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityTypeManager;
+use Drupal\Core\Entity\FieldableEntityInterface;
+use Drupal\Core\Entity\TranslatableInterface;
+use Drupal\Core\Field\EntityReferenceFieldItemListInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\graphql\Plugin\GraphQL\DataProducer\DataProducerPluginBase;
+use Drupal\graphql\GraphQL\Buffers\EntityRevisionBuffer;
+use GraphQL\Deferred;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Loads the entity reference revisions.
+ *
+ * @DataProducer(
+ *   id = "entity_reference_revisions",
```

Comment:
This field (entity_reference_revisions) is from a contrib module so it should use the "provider" annotation to only enable it when that module is available.

Reply:
Added -> https://github.com/drupal-graphql/graphql/pull/909/files#diff-e1711fdf60230bcab5de56c6c49d9279R24

Please re-check